Algorithm.CSharp/AddRemoveOptionUniverseRegressionAlgorithm.cs

```diff
@@ -110,20 +111,21 @@ public override void OnData(Slice slice)
                 googOptionChain.SetFilter(u =>
                 {
                     // we added the universe at 10, the universe selection data should not be from before
-                    if (u.Underlying.EndTime.Hour < 10)
+                    if (u.LocalTime.Hour < 10)
                     {
-                        throw new RegressionTestException($"Unexpected underlying data point {u.Underlying.EndTime} {u.Underlying}");
+                        throw new RegressionTestException($"Unexpected selection time {u.LocalTime}");
                     }
                     // find first put above market price
                     return u.IncludeWeeklys()
                         .Strikes(+1, +3)
                         .Expiration(TimeSpan.Zero, TimeSpan.FromDays(1))
-                        .Contracts(c => c.Where(s => s.ID.OptionRight == OptionRight.Put));
+                        .Contracts(c => c.Select(x => x.Symbol).Where(s => s.ID.OptionRight == OptionRight.Put));
```

Comment:
We don't need `.Select(x => x.Symbol)` given `ID` is on the OptionUniverse data too?